app/services/profiles/import_alerts_and_personal_care_needs.rb

```diff
@@ -0,0 +1,13 @@
+module Profiles
+  class ImportAlertsAndPersonalCareNeeds
+    def self.call(profile, prison_number)
+      alerts = NomisClient::Alerts.get([prison_number]).group_by { |p| p.fetch(:offender_no) }
+      personal_care_needs = NomisClient::PersonalCareNeeds.get([prison_number]).group_by { |p| p.fetch(:offender_no) }
+
+      Alerts::Importer.new(profile: profile, alerts: alerts.fetch(prison_number, [])).call
+      PersonalCareNeeds::Importer.new(profile: profile, personal_care_needs: personal_care_needs.fetch(prison_number, [])).call
+
+      profile
```

Comment:
```suggestion
      profile.save
```

We currently don't save the attributes coming from Nomis, we just assign them:

https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/models/profile.rb#L33

That's one of the misnomer consequences of the importer services. They don't actually import anything 😢

Reply:
See also https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/services/moves/import_people.rb#L35-L38

Reply:
```suggestion
      profile.save!
```

Reply:
Good point! I'll update the tests since the function now returns true/false.